### PR TITLE
Updated LImageOverlay

### PR DESCRIPTION
**Pull request by jericopulvera**
I added the missing options for Image Overlay then I also added the bringToFront() and bringToBack() methods.

### CODE REVIEW

**Comment by lordfuoco** [comment]
@jericopulvera Hi thank you for the good work, could you please resolve the conflicts ?

**Comment by jericopulvera** [comment]
@lordfuoco I added all the methods that I saw in leaflets latest documentation. https://leafletjs.com/reference-1.3.4.html#imageoverlay

Not really sure if I shouldn't have included the following methods since Vue data binding can handle.
* setOpacity
* setUrl
* setBounds
* setZIndex

**Comment by lordfuoco** [comment]
@jericopulvera Thank you!
2 small notes: 
-  "leaflet": "1.3.1" we locked the version to this for now
- You did well to add those methods but I would update the related props to have the flag: custom: true

Doing so the propsBinder function will connect the props to those methods and enable reactivity.

File: src/components/LImageOverlay.vue [hunk 1]
```diff
@@ -85,9 +90,6 @@ export default {
     setBounds (bounds) {
       return this.mapObject.setBounds(bounds)
     },
-    setZIndex (zIndex) {
-      return this.mapObject.setBounds(zIndex)
-    },
```
Comment (lordfuoco) [inline src/components/LImageOverlay.vue]: Why remove this method?

**Comment by jericopulvera** [comment]
@lordfuoco It was not included at leflet 1.3.1

**Pull request merged**